packages/block-editor/src/components/image-size-control/index.js

```diff
@@ -39,7 +34,7 @@ export default function ImageSizeControl( {
 
 	return (
 		<>
-			{ ! isEmpty( imageSizeOptions ) && (
+			{ imageSizeOptions && imageSizeOptions.length > 0 && (
```

Comment:
`SelectControl`'s options is always an array. However, we're just making sure it's not nullish before actually checking the array length.